ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/XARecoveryModuleUnitTest.java

```diff
@@ -525,20 +533,17 @@ public XAResource[] getXAResources() throws Exception {
         // 1st pass: returns one xid (count is 1)
         xarm.periodicWorkFirstPass();
         // 2nd pass: throws an exception (count is 2)
```

Comment:
These comments are no longer valid because the second pass does not call recover(). Also the following comments stating the value of count are no longer valid. And instead of incrementing `recoverCount` by 2 each time sometimes it will only increment by 1 or zero if exceptions are thrown. Also it might be worthwhile using `recoverCount` instead of `count` in the dummy XAResource so that the test can continue asserting that the recover calls are called the correct number of times.

Reply:
I think this test is now very confusing and may be deleted. However I will try to change it.